README.md

```diff
@@ -13,10 +13,10 @@ https://pkg.contrastsecurity.com/go-agent-release/.
 
 ## System Requirements
 
-* go1.17 or later, which can be downloaded from https://go.dev/dl/.
+* go1.21 or later, which can be downloaded from https://go.dev/dl/.
 
 > **Note**
-> While `contrast-go-installer` works with version 1.17 and on, `contrast-go`
```

Comment:
Double checking - does #33 mean that 1.21 is now required?

Reply:
Actually, it looks like if we want to stay up to date, we should set our `go.mod` to `1.22`. I chose to update it to `1.21` since that's what our agent `go.mod` is, but I could see upgrading to `1.22`.

Reply:
It's a little unfortunate because that's just a test dependency. I'm inclined to suggest we decline that PR just so that we can keep working with older versions of Go -- that way users get the version requirements from us directly, rather than seeing an error message from Go that we don't really control when trying to run the installer.